### PR TITLE
Add docker support

### PR DESCRIPTION
**Pull request by simon511000**

### CODE REVIEW

**Comment by Liritt** [comment]
ports:
  - "5432:5433"
environment:
  POSTGRES_DB: bookmarker
  POSTGRES_USER: root
  POSTGRES_PASSWORD: root

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+  bookmarker:
+    build: .
+    volumes:
+      - ./settings.py:/app/settings.py:ro
+    depends_on:
+      - postgres
+  postgres:
+    image: postgres:15
+    environment:
+      POSTGRES_PASSWORD: password
```
Comment (Liritt) [inline docker-compose.yml:11]:
```suggestion
    environment:
      POSTGRES_PASSWORD: password
```
```
ports:
  - "5432:5433"
environment:
  - POSTGRES_DB: bookmarker
  - POSTGRES_USER: root
  - POSTGRES_PASSWORD: root
```

Comment (Liritt) [inline docker-compose.yml:11]:
```suggestion
ports:
  - "5432:5433"
environment:
  - POSTGRES_DB: bookmarker
  - POSTGRES_USER: root
  - POSTGRES_PASSWORD: root
```

Comment (simon511000) [reply docker-compose.yml:11]: There is an indentation problem

**Pull request merged**